app/utils/CheckYourAnswersHelper.scala

```diff
@@ -28,10 +28,14 @@ import models._
 import pages._
 import play.api.i18n.Messages
 import play.twirl.api.{Html, HtmlFormat}
+import queries.{ClaimDateQuery, ClaimIdQuery}
 import viewmodels.{AnswerRow, AnswerSection}
+import views.DateTimeFormats
 
 class CheckYourAnswersHelper(userAnswers: UserAnswers)(implicit messages: Messages) {
```

Comment:
Summary of changes:
- made all of the "helper" methods private except for the two client methods - `getCheckYourAnswerSections` and `getAmendCheckYourAnswerSections` (move these to top)
- added new method `getCreateConfirmationSections` to create a read-only (without change urls) version of "CYA sections" with the addition of an "Application" section at the start containing case ref id and submission date/time